v0/util/index.js

```diff
@@ -1322,21 +1322,26 @@ class CustomError extends Error {
  * @param {*} destination
  * @param {*} transformStage
  */
-function generateErrorObject(error, destination, transformStage) {
-  // check err is object
-  // filter to check if it is coming from cdk
-  if (error.fromCdk) {
-    return error;
-  }
+function generateErrorObject(error, destination = "", transformStage) {
   const { status, message, destinationResponse } = error;
   let { statTags } = error;
   if (!statTags) {
     statTags = {
-      destination,
+      destType: destination,
       stage: transformStage,
       scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.EXCEPTION.SCOPE
     };
   }
+  // In previous versions of cdk, we had destination tag
+  // This block is mainly to support it and also for backward for non-cdk destinations
```

Comment:
Backward compatibility for non-CDK destinations?

Reply:
This is present as protection in-case I missed changing the `destination` tag to `destType`

Reply:
Lol...we should identify all and fix them.

Reply:
Fixed them for current destinations inside implemented without CDK